spec/convertion_by_format/check_open_docx_by_screen_spec.rb

```diff
@@ -4,7 +4,7 @@
 FileHelper.clear_dir('files_tmp')
 palladium = PalladiumHelper.new(DocumentServerHelper.get_version, 'Convert DOCX')
 result_sets = palladium.get_result_sets(StaticData::POSITIVE_STATUSES)
-files = s3.get_files_by_prefix('docx')
+files = s3.files_from_folder 'docx'
```

Comment:
I think in all cases is better to leave parentheses
So it'll be 
`files = s3.files_from_folder('docx')`
We usually use them in 100% cases but there is no rule for rubocop for check it (